app/questionnaire/questionnaire_store_updater.py

```diff
@@ -74,7 +80,9 @@ def remove_completed_relationship_locations_for_list_name(
             for target in target_relationship_collectors:
                 block_id = target["id"]
                 section_id = self._schema.get_section_for_block_id(block_id)["id"]  # type: ignore
-                self.remove_completed_location(Location(section_id, block_id))
+                self._progress_store.remove_completed_location(
+                    Location(section_id, block_id)
+                )
```

Comment:
We should either use the `self.remove_completed_location` wrapper, or remove it. I don't mind as long as it is consistent.

Reply:
I'd removed the wrapper from the base updater, its only in the subclass, as theres multiple places still using the wrapper for the normal QSU, and I can't keep it in the base one due to lack of location - could look at updating callers but it is consistent within each class if thats enough? otherwise will look at passing in location to any callers

Reply:
OK cool, if its just 3 places, I'll pass it in and keep & use the wrapper without the default 👍